File: app/src/main/java/ru/samtakoy/core/api/network/pojo/RemoteFilesInfo.java
package ru.samtakoy.core.api.network.pojo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RemoteFilesInfo {

    @SerializedName("files")
    @Expose
    private List<RemoteFile> files = null;

    public List<RemoteFile> getFiles() {
        return files;
    }

    public void setFiles(List<RemoteFile> files) {
        this.files = files;
    }

}

File: app/src/main/kotlin/ru/samtakoy/core/data/network/pojo/RemoteFilesInfo.kt
package ru.samtakoy.core.data.network.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RemoteFilesInfo {
    @SerializedName("files")
    @Expose
    val files: List<RemoteFile>? = null
}
